keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/CombinedRepository.kt

```diff
@@ -298,8 +301,8 @@ class CombinedRepository(
   override fun getResourcesByApplication(application: String): List<Resource<*>> =
     resourceRepository.getResourcesByApplication(application)
 
-  override fun getSummaryByApplication(application: String): List<ResourceSummary> =
-    resourceRepository.getSummaryByApplication(application)
+  override fun getResourceSummaries(deliveryConfig: DeliveryConfig): List<ResourceSummary> =
```

Comment:
Changed from a lookup by application name to a lookup by delivery config to match all the other `getXyzSummaries()` functions, and because the delivery config is necessary to find associations between resources and artifacts.